src/render/locals.rs

```diff
@@ -374,7 +387,14 @@ pub fn print_alloc(
                 .is_range_defined(Size::from_bytes(i), Size::from_bytes(i + 1))
                 .is_ok()
             {
-                write!(&mut s, "{:02x}", alloc.inspect_with_undef_and_ptr_outside_interpreter(i as usize .. i as usize + 1)[0]).unwrap();
+                write!(
+                    &mut s,
+                    "{:02x}",
+                    alloc
+                        .inspect_with_undef_and_ptr_outside_interpreter(i as usize..i as usize + 1)
+                        [0]
+                )
+                .unwrap();
```

Comment:
And this

Reply:
The `alloc.inspect_with_undef_and_ptr_outside_interpreter` could be extracted into a variable. (`byte`?)